packages/aws-cdk/lib/api/bootstrap/bootstrap-template.json

```diff
@@ -337,7 +337,8 @@
                     "s3:GetObject*",                    "s3:GetBucket*",
                     "s3:List*",                         "s3:Abort*",
                     "s3:DeleteObject*",                 "s3:PutObject*",
-                    "kms:Decrypt",                      "kms:DescribeKey"
+                    "kms:Decrypt",    "kms:DescribeKey",     "kms:Encrypt",
```

Comment:
I would like the reasons we add each of these permissions to be documented somewhere in-line. Can you think of a good place? (Unfortunately JSON doesn't allow comments... Sids?)